Audiobook Player/PlayerViewController.swift

```diff
@@ -444,7 +426,16 @@ extension PlayerViewController: AVAudioPlayerDelegate {
     
     @objc func bookEnd() {
         self.playButton.setImage(self.playImage, for: UIControlState())
-        self.requestReview()
+        self.requestReview()        
+    }
+
+    @objc func bookChange(_ notification:Notification) {
+        guard let userInfo = notification.userInfo,
+            let books = userInfo["books"] as? [Book],
+            let book = books.first else {
+                return
+        }
+        setupView(book: book)
```

Comment:
don't forget to update the current book variable `self.currentBook = book`. Right now after the book changes, the timer won't update